src/algorithms/standard/minmax.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2006-2016  Music Technology Group - Universitat Pompeu Fabra
+ *
+ * This file is part of Essentia
+ *
+ * Essentia is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation (FSF), either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the Affero GNU General Public License
+ * version 3 along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+
+#include "MinMax.h"
+#include "essentiamath.h"
+#include <sstream>
+
+using namespace essentia;
+using namespace standard;
+
+const char* MinMax::name = "MinMax";
+const char* MinMax::category = "Standard";
+const char* MinMax::description = DOC("This algorithm calculates the minimum or maximum value of an array.");
```

Comment:
There may be more than one minimum or maximum value in the array. Please, specify the behavior in such a case in the DOC. For example, we use `std::max_element` and it specified that "If several elements in the range are equivalent to the greatest element, returns the iterator to the first such element."